### DISCUSSION TITLE
Reinstate Classic User Account Control from Vista-Windows 10 RTM?

### DISCUSSION THREAD

**Discussion by TheDolphinSlayer** [valinet/ExplorerPatcher#247] [Ideas]
More curious if this can be achieved with ExplorerPatcher(though UAC is a different thing)

also an extremely dumb question considering this is literally made for Windows 11 but~ does this work on Windows 10?

**Comment by valinet** [comment]
1. ExplorerPatcher is not some magic potion that enables hidden/old/deprecated stuff. I appreciate the comparison, but in reality, every feature has to have someone actually code it in, study how it’s done etc. So yeah, I don’t know how that is done, where that lived etc, never studied it, so without me putting in some code on this side, it cannot automatically do it, it doesn’t write itself up.
2. What should it do in Windows 10? The purpose of EP is already served in there. Some stuff which was the same should technically work, but I haven’t even tried running it on Windows 10. Maybe try it in a VM, if you feel like doing it, and report back your findings.
